src/apps/competitions/migrations/0024_auto_20201029_2210.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.2.13 on 2020-10-29 22:10
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('competitions', '0023_auto_20201020_0038'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='submission',
+            name='is_private',
+            field=models.BooleanField(default=False),
+        ),
+        migrations.AddField(
+            model_name='submission',
+            name='is_specific_task_re_run',
+            field=models.BooleanField(default=False),
+        ),
+    ]
```

Comment:
There's a 0024 migration already on develop. Need to bump this.

Reply:
👌🏼